src/pages/credit-invoice/Forms/CreditInvoiceForm.js

```diff
@@ -740,6 +738,14 @@ export default function CreditInvoiceForm({ _labels, access, recordId, plantId,
       width: 130
     }
   ]
+  async function getCurrencyGoc(currencyId) {
+    const res = await getRequest({
+      extension: RemittanceSettingsRepository.CorrespondentCurrency.get,
+      parameters: `_corId=${formik.values.corId}&_currencyId=${currencyId}`
+    })
+
+    return res?.record?.goc
```

Comment:
return res?.record and then get the goc value in the update function. if the res.record is null then you can consider that goc is false